### ISSUE TITLE
Can't use numpy_to_nx

### ISSUE THREAD

**Issue by elfprince13** [danielegrattarola/spektral#15]
```python
import spektral
import numpy as np
labels = np.array([[1,2,3],[4,5,6]])
adj = np.array([[0,1],[1,0]])
spektral.utils.numpy_to_nx(adj,labels)
```
results in:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/thomas/Library/Python/2.7/lib/python/site-packages/spektral/utils/conversion.py", line 192, in numpy_to_nx
    nx.set_node_attributes(g, node_attrs, nf_name)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/networkx/classes/function.py", line 660, in set_node_attributes
    G.nodes[n].update(d)
TypeError: cannot convert dictionary update sequence element #0 to a sequence
```

Using Python 2.7.16. `pip` reports Keras 2.2.5, tensorflow 1.14.0, and spektral 0.0.13.

**Comment by elfprince13** [comment]
This should *mostly* be okay, since eventually everything needs to be in numpy format anyway, but it will make serialization and deserialization moderately more annoying.

**Comment by danielegrattarola** [comment]
Thanks for spotting this. It should be fixed by commit 66ff5ca0621f9d01d117cdfc9aad9d6899ed86e3, tested with the following code:

```python
import spektral
import numpy as np

nf = np.ones((2, 3))
ef = np.ones((2, 2, 3))
adj = np.ones((2, 2))
spektral.utils.numpy_to_nx(adj, nf, ef)
```

As a side note, Spektral is not guaranteed to work with Python 2 because I only test for Python 3. Just a heads-up should you run into problems in the future.

Cheers
